_dactyl/dactyl-config.yml

```diff
@@ -41,36 +41,56 @@ pages:
         targets:
             - en
 
-    -   name: Getting Started
-        funnel: Docs
-        doc_type: Overview
-        html: getting-started.html
-        md: getting-started.md
-        template: template-doc.html
-        blurb: Start an Interledger node locally and create a local ILP test network.
-        targets:
-            - en
-
-    -   name: Send Value Programmatically
+    -   name: Set Up and Use Interledger Accounts With a User Interface
```

Comment:
Should we shorten this title (and the one for the other article) since it might be too long for the sidebar on smaller screens?